client/extensions/woocommerce/app/index.js

```diff
@@ -19,6 +19,7 @@ import config from 'config';
 import DocumentHead from 'components/data/document-head';
 import { fetchSetupChoices } from 'woocommerce/state/sites/setup-choices/actions';
 import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteHomeUrl } from 'state/sites/selectors';
```

Comment:
I wonder if we should call this a `baseUrl` rather than home, as sometimes it isn't home...

Reply:
It seems that `baseUrl` in Calypso often refers to the first part of a url that had path and/or query params added to it. @pablinos was suggesting `getSiteAdminHomePageUrl`, but `adminUrl` refers to wp-admin in another site selector.

`getSiteDefaultUrl`... `getSiteLandingUrl`... `getDefaultSitePage`?

Reply:
Any of the above look good. Maybe `getDefaultSitePage`?

Reply:
`getMySitesDefaultUrl`?

Reply:
I like using `MySites` to differentiate it from the other site selectors that get site properties, like `getSiteSlug`. Ended up with `getMySitesDefaultPage`. PR here: https://github.com/Automattic/wp-calypso/pull/36899